spring-data-jpa/src/test/java/org/springframework/data/jpa/repository/UserRepositoryTests.java

```diff
@@ -15,23 +15,14 @@
  */
 package org.springframework.data.jpa.repository;
 
-import static java.util.Arrays.asList;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.springframework.data.domain.Example.of;
-import static org.springframework.data.domain.ExampleMatcher.GenericPropertyMatcher;
-import static org.springframework.data.domain.ExampleMatcher.StringMatcher;
-import static org.springframework.data.domain.ExampleMatcher.matching;
-import static org.springframework.data.domain.Sort.Direction.ASC;
-import static org.springframework.data.domain.Sort.Direction.DESC;
+import static java.util.Arrays.*;
```

Comment:
The import reformatting indicates a different threshold for `*` imports. Please revisit your formatter settings.

Reply:
I have my settings matching https://github.com/spring-projects/spring-data-build/tree/main/etc/ide.

Is there a typo in that doc, such that BOTH fields should have `10`? Because when I put in `10` to BOTH settings, this file switches back to what you are indicating here.

Reply:
Please ignore my comment. I missed the `static`.